src/generic/grideditors.cpp

```diff
@@ -228,10 +228,17 @@ void wxGridCellEditorEvtHandler::OnChar(wxKeyEvent& event)
 // wxGridCellEditor
 // ----------------------------------------------------------------------------
 
-wxGridCellEditor::wxGridCellEditor()
+wxGridCellEditor::wxGridCellEditor(const wxGridCellEditor& other)
+    : wxGridCellWorker(other),
+      m_control(other.m_control)
 {
-    m_control = NULL;
-    m_attr = NULL;
+    if ( other.m_attr != NULL )
+    {
+        m_attr = other.m_attr->Clone();
+    }
+    m_colFgOld = wxColour(other.m_colFgOld);
+    m_colBgOld = wxColour(other.m_colBgOld);
+    m_fontOld = wxFont(other.m_fontOld);
```

Comment:
All those need should be initialized in the initializer list and without making unnecessary copies.